### PR TITLE
Initial version of Campaign Requests with plain CRUD functionality

### PR DESCRIPTION
**Pull request by jwerak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+  - Flask-Migrate
+- Docker
+- PostgreSQL
+
+### Dev machine setup
+
+Activate virtualenv 
+
+``` bash
+./bin/activate
```
Comment (agiertli) [inline README.md]: I was not able to run this app due to different dependencies (python3, virtualenv, flusk..)
Would it be possible to properly containerise the app?

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+# On changed schema
+python3 manage.py db migrate
+python3 manage.py db upgrade
+```
+
+### Cleanup
+
+```bash
+docker-compose down --volumes
+```
```
Comment (agiertli) [inline README.md:234]: Please add example CRUD requests (including payload) for all the basic operations.
Me and Tomas are using httpie [1] in our examples.

[1] https://httpie.org/

File: init-db.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+set -e
```
Comment (agiertli) [inline init-db.sh]: Shouldn't this call the sql ddl script so the DB would be initialised automatically during the container start up?

File: README.md (hunk 1 above)
Comment (jwerak) [reply README.md]: yep, good feedback, I've added simple dockerfile and setup is now just `docker-compose up`

File: README.md (hunk 2 above)
Comment (jwerak) [reply README.md:234]: added.

File: init-db.sh (hunk 3 above)
Comment (jwerak) [reply init-db.sh]: It should have been initially. 
I've moved it now as part of api startup script.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,8 @@
+FROM python:3
```
Comment (rwngwn) [inline Dockerfile]: UBI please

File: config.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+app.config["SQLALCHEMY_ECHO"] = True
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get("DATABASE_URL")
+db = SQLAlchemy(app)
+
+# Initialize Marshmallow
+ma = Marshmallow(app)
+
+migrate = Migrate(app, db)
+import models
```
Comment (rwngwn) [inline config.py]: import as last statement?

File: include/python3.7m [hunk 6]
```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/include/python3.7m
```
Comment (rwngwn) [inline include/python3.7m]: whats that ?

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,35 @@
+alembic==1.0.10
+botocore==1.12.172
+certifi==2019.6.16
+chardet==3.0.4
+Click==7.0
```
Comment (rwngwn) [inline requirements.txt]: why click? this is for command line apps or not?

File: start.sh [hunk 8]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Create/Update database schema
+flask db upgrade
```
Comment (rwngwn) [inline start.sh]: cannot this be handled in the python code?

**Review by rwngwn — CHANGES_REQUESTED** [review]

File: include/python3.7m (hunk 6 above)
Comment (jwerak) [reply include/python3.7m]: ugly thing, removed...

File: requirements.txt (hunk 7 above)
Comment (jwerak) [reply requirements.txt]: removed.

**Review by rwngwn — APPROVED** [review]

**Pull request merged**